src/components/ebay-pagination/component.js

```diff
@@ -93,7 +116,18 @@ export default {
             }
         }
 
-        return { start, end, hideDots, dotsIndex };
+        if (hasLeadingDots) {
```

Comment:
This logic is a bit hard to follow. I think some comments would help.

Reply:
The plan is to clean this part up in the tech debt.